pkg/netlink/fake/netlink.go

```diff
@@ -342,3 +352,11 @@ func (n *NetLink) AwaitNoRule(table int) {
 		return n.getRule(table)
 	}, 5).Should(BeNil(), "Rule for %v exists", table)
 }
+
+func (n *NetLink) VerifyMtuProbe(mtuProbe string) {
+	Expect(mtuProbe).To(Equal(n.mtuProbe))
+}
+
+func (n *NetLink) VerifyBaseMss(baseMss string) {
+	Expect(baseMss).To(Equal(n.baseMss))
+}
```

Comment:
Since we set these atomically we should really verify them atomically and also assume multi-threaded. 

```suggestion
func (n *NetLink) VerifyTCPMTUProbe(mtuProbe, baseMss string) {
        format := "MTU probe: %s, Base MSS: %s"
	Eventually(func() string {
            n.Lock()
            defer n.Unlock()
	    
             return fmt.Sprintf(format, n.mtuProbe, n.baseMss)
	}, 5).Should(Equal(fmt.Sprintf(format, mtuProbe, baseMss)))
}
```

Reply:
Done.